src/toGamut.js

```diff
@@ -104,3 +113,103 @@ export default function toGamut (color, {method = defaults.gamut_mapping, space
 }
 
 toGamut.returns = "color";
+
+// The reference colors to be used if lightness is out of the range 0-1 in the
+// `Oklch` space. These are created in the `Oklab` space, as it is used by the
+// DeltaEOK calculation, so it is guaranteed to be imported.
```

Comment:
I'd use a reference to the color space then. Using string ids requires color space registration, which breaks tree-shaking. Can't remember if we do something special for basic color spaces like this one, but I don't think so,